app/models/requests/requestable_decorator.rb

```diff
@@ -125,7 +125,12 @@ def status_badge
                   else
                     "badge-success"
                   end
-      content_tag(:span, requestable.status_label, class: "availability--label badge #{css_class}")
+      status = if requestable.status_label.nil?
+                 requestable.status
+               else
+                 requestable.status + ' - ' + requestable.status_label
```

Comment:
@sdellis If status_label is empty can we not print the "-"

Reply:
Actually isn't status_label the one that is a boolean (Available|Not Available)? That one should be first. Followed by the more specific value. So it would show up as

Not Available - Lost for lost items.